pep-0688.rst

```diff
@@ -1,12 +1,14 @@
 PEP: 688
 Title: Making the buffer protocol accessible in Python
 Author: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Discussions-To: https://discuss.python.org/t/15265
 Status: Draft
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 23-Apr-2022
 Python-Version: 3.12
-
+Post-History: `23-Apr-2022 <https://mail.python.org/archives/list/typing-sig@python.org/thread/CX7GPSIYQEL23RXMYL66GAKGP4RLUD7P/>`__,
+              `25-Apr-2022 <https://discuss.python.org/t/15265>`__
 
```

Comment:
Keep two blank lines per style
```suggestion


```

Reply:
It also helps avoid any ambiguity when regex parsing